rundoc/parsers.py

```diff
@@ -17,8 +17,10 @@ def mkd_to_html(mkd, tags='', must_have_tags='', must_not_have_tags=''):
     html_data = markdown.markdown(
         mkd,
         extensions = [ 
-            'markdown_rundoc.rundoc_code(tags={},must_have_tags={}, must_not_have_tags={})'.format(
-                tags, must_have_tags, must_not_have_tags,
```

Comment:
This was causing the following warning:
tests/test_unit.py::test_parsers__parse_output
  /home/pmandic/repos/nul.one/rundoc/env/lib/python3.5/site-packages/markdown/__init__.py:222: DeprecationWarning: Setting configs in the Named Extension string is deprecated. It is recommended that you pass an instance of the extension class to Markdown or use the "extension_configs" keyword. The current behavior will raise an error in version 2.7. See the Release Notes for Python-Markdown version 2.6 for more info.
    '2.6 for more info.', DeprecationWarning)